prov/verbs/src/verbs_info.c

```diff
@@ -113,6 +113,15 @@ const struct fi_rx_attr verbs_rx_attr = {
 	.total_buffered_recv	= 0,
 };
 
+const struct fi_rx_attr verbs_ro_rx_attr = {
+	.caps			= VERBS_MSG_RX_CAPS,
+	.mode			= VERBS_RX_MODE,
+	.op_flags		= FI_COMPLETION,
+	.msg_order		= 0,
+	.comp_order		= FI_ORDER_STRICT | FI_ORDER_DATA,
```

Comment:
I think we need to drop FI_ORDER_DATA, since the data within a message may be placed into memory OOO